ci-operator/step-registry/openstack/install/createcluster/openstack-install-createcluster-commands.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+if [[ -z "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
+  echo "OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is an empty string, exiting"
+  exit 1
+fi
+
+echo "Installing from release ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME}/${BUILD_ID}
+export HOME=/tmp
+
+case "${CLUSTER_TYPE}" in
+openstack) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+openstack-vexxhost) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+*) >&2 echo "Unsupported cluster type '${CLUSTER_TYPE}'"
+esac
+
+ASSETS_DIR=/tmp/installer
+mkdir "${ASSETS_DIR}/"
+cp "${SHARED_DIR}/install-config.yaml" "${ASSETS_DIR}/"
+
+mkdir -p ~/.ssh
+cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
+
+
+if [[ "${USE_ETCD_RAMDISK}" == "true" ]]; then
+  TF_LOG=debug openshift-install --dir=${ASSETS_DIR} create ignition-configs --log-level=debug
+  python -c \
+      'import json, sys; j = json.load(sys.stdin); j[u"systemd"] = {}; j[u"systemd"][u"units"] = [{u"contents": "[Unit]\nDescription=Mount etcd as a ramdisk\nBefore=local-fs.target\n[Mount]\n What=none\nWhere=/var/lib/etcd\nType=tmpfs\nOptions=size=2G\n[Install]\nWantedBy=local-fs.target", u"enabled": True, u"name":u"var-lib-etcd.mount"}]; json.dump(j, sys.stdout)' \
+      <${ASSETS_DIR}/master.ign \
+      >${ASSETS_DIR}/master.ign.out
+  mv ${ASSETS_DIR}/master.ign.out ${ASSETS_DIR}/master.ign
+fi
```

Comment:
Could you perhaps submit this change to `ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh` so that every platform can benefit from it and we reduce code duplication?

At least `kubevirt` and `ovirt` platforms have the same ramdisk need.

Reply:
yes. Let me start that process there as well. 
But we might want to have our own install steps for one off cases, like this one. 
ipi-install-install does not allow for easy manipulation of any of the resource files, and we do that sort of thing all the time in openstack installation. 
I was thinking we can start things in this step and migrate them to ip-install-install if they are generic.

Reply:
once it merges there I'll abandon this PR